src/index.html

```diff
@@ -1,12 +1,302 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="logo">
+          <a href="#home" class="logo__link">
+            <img src="./images/logo.png"
+            alt="logo"
+            class="logo__img"
+            >
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__item">
              <a href="#apple" class="nav__link is-active">Apple</a>
            </li>
            
            <li class="nav__item">
              <a href="#samsung" class="nav__link">Samsung</a>
            </li>
```
Add blank line between multiple sibling blocks of html